actions/main/src/main.sh

```diff
@@ -100,20 +92,11 @@ configureCredentials () {
   else
     echo "Skipping importing environment vars for configureCredentials"
   fi
-  if [[ "$VAULT_TOKEN" != "" ]]; then
-    echo "Vault token already set skipping configureCredentials function"
-  elif [[ "${role_id}" != "" ]] && [[ "${secret_id}" != "" ]] && [[ "${vault_address}" != "" ]]; then
-    export VAULT_ADDR=${vault_address}
-    export VAULT_TOKEN=$(curl \
-      --request POST \
-      --data '{"role_id":"'"${role_id}"'","secret_id":"'"${secret_id}"'"}' \
-      ${vault_address}/v1/auth/approle/login | jq -r .auth.client_token)
-      echo "export VAULT_TOKEN=${VAULT_TOKEN}" >> env_vars
-    /usr/local/bin/vault read -format=json secret/dsde/datarepo/dev/sa-key.json | \
-      jq .data > ${GOOGLE_APPLICATION_CREDENTIALS}
+  if [[ "$SA_B64_CREDENTIALS" != "" ]]; then
```

Comment:
Is this correct? I would expect this to be `INPUT_SA_B64_CREDENTIALS`
```suggestion
  if [[ "$INPUT_SA_B64_CREDENTIALS" != "" ]]; then
```

Reply:
That's a good catch ... I think I'm missing something in the optional inputs section of the script that would set that value. It looks like that's where things get exported with `INPUT_` prefixes. I am not sure how to test that - will have a look.